views/chitter.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
i tried to connect a css using the <link> but I hadn't included all the information in here